mocap_optitrack_driver/launch/optitrack2.launch.py

```diff
@@ -61,19 +61,19 @@ def generate_launch_description():
     )
 
     # Make the driver node take the 'activate' transition
-    # driver_activate_trans_event = EmitEvent(
```

Comment:
I need my driver to activate itself. hence there is this autostart event actiavatedi n the launch file

Reply:
Done.